### ISSUE TITLE
Hanging on "Validating existing repo" when adding an existing repo in the Flatpak version

### ISSUE THREAD

**Issue by mhoeher** [borgbase/vorta#603]
**Describe the bug**
Vorta waits forever on "Validating existing repo..." when adding back an existing repository on a server connected to via SSH. This happens for me in the Flatpak version of the app. Looking at the task manager, this is what the app seems to wait for:

```bash
/usr/bin/python3 /app/bin/borg info --info --json --log-json user@server:/path/to/backup/location
```

When I run the same command on the shell, it returns immediately.

**To Reproduce**
1. Start the Flatpak version of Vorta.
2. Add an existing repository URL.
3. Press "Add".
4. Vorta now waits forever, showing "Validating existing repo..."
5. Addon: If I click "Cancel", the app crashes

**Desktop (please complete the following information):**
 - OS: Fedora 32

**Additional context**

If it helps, this is the output when running `flatpak run ...`:

```
QIBusPlatformInputContext: invalid portal bus.
2020-08-30 19:05:41,896 - vorta.i18n - DEBUG - Loading translation failed for ['en', 'en-US', 'en-Latn-US'].
2020-08-30 19:05:41,933 - apscheduler.scheduler - INFO - Scheduler started
2020-08-30 19:05:41,997 - vorta.borg.borg_thread - INFO - Running command /app/bin/borg --version
2020-08-30 19:06:15,137 - vorta.borg.borg_thread - DEBUG - Using VortaSecretStorageKeyring keyring to store passwords.
2020-08-30 19:06:15,140 - asyncio - DEBUG - Using selector: EpollSelector
2020-08-30 19:06:15,156 - root - DEBUG - Found 0 passwords matching repo URL.
2020-08-30 19:06:15,157 - vorta.borg.borg_thread - DEBUG - Password not found in primary keyring. Falling back to VortaDBKeyring.
2020-08-30 19:06:15,192 - vorta.borg.borg_thread - INFO - Running command /app/bin/borg info --info --json --log-json user@server:/path/to/backup/location
QThread: Destroyed while thread is still running
```

When installing the app via pip (`pip install --user vorta`) the issue does not occur.

**Comment by Hofer-Julian** [comment]
Which desktop environment do you use?

**Comment by mhoeher** [comment]
I am running KDE, installed form the Fedora repositories.

**Comment by Hofer-Julian** [comment]
So you've downloaded https://spins.fedoraproject.org/kde/?

Or did you install KDE via the package manager?

**Comment by mhoeher** [comment]
IIRC (it is a few years back) I installed the KDE spin, but technically this should be the same as installing KDE from the Fedora repositories.

**Comment by Hofer-Julian** [comment]
Yeah, the reason I am asking this is because of this part of your log

QIBusPlatformInputContext: invalid portal bus.

It seems like your portals are not working. 

Do you have problems with other flatpaks as well?

**Comment by mhoeher** [comment]
I didn't notice anything suspicious until now. I played around with IBus and it's Qt integration some time back, maybe this is just a remainder of this. I could try to check this on the machine this evening.

**Comment by Hofer-Julian** [comment]
If you have time, it would be great if you could check if this problem is reproducable in a virtual machine with Fedora KDE spin

**Comment by samuel-w** [comment]
I've experienced this issue as well, but its happens with no pattern, so this is going to be a pain to fix.

**Comment by samuel-w** [comment]
Check and see if it is fixed by 0.7.1

**Comment by mhoeher** [comment]
Hey, sorry for reacting only that late. I was quite busy and did not find time to look into this anytime sooner.

Anyway... I upgraded to v0.7.1 and with this version it now works perfectly :partying_face: So, thanks a lot!
